lib/validators/movie.js

```diff
@@ -7,6 +7,10 @@ exports.payload = Joi.object().keys({
   release_year: Joi.number().integer().min(1878).max(9999).optional()
 });
 
+exports.payload_location = Joi.object().keys({
```

Comment:
nit: `payloadLocation`. We only really use snake-case (underscores) things returned from a serializer because that's the better convention for JSON.